### PR TITLE
Add docs for Import Postman Collection feature

### LINKED ISSUE #409: Migrating Postman Collections
Update this topic:
https://meta.stoplight.io/docs/platform/ZG9jOjYzMTA0OA-migrating-from-postman

With information from this issue: 
https://github.com/stoplightio/platform-internal/pull/11707

### PR DESCRIPTION
**Pull request by heitortsergent**
Closes #409.

- [ ] Rename and move document to new section
- [ ] Add option to other pages in the docs
  - [x] https://meta.stoplight.io/docs/platform/0e6a3688afcc3-quickstart-guide
  - [x] https://meta.stoplight.io/docs/platform/58bf119ddff88-work-with-files#directory-structure

### CODE REVIEW

File: 6.-migrations/postman.md [hunk 1]
```diff
@@ -1,110 +1,28 @@
-# Migrating from Postman
+# Import Postman Collection
 
-## Introduction
+Stoplight enables users to import a [Postman Collection](https://learning.postman.com/docs/sending-requests/intro-to-collections/) file from their local computer directly into Stoplight Studio.
 
-Stoplight Platform supports all sorts of workflows for API design and documentation: Design-First and Code-First being the most common. Design First suggest teams are creating API descriptions in formats like OpenAPI and JSON Schema, and writing code that corresponds to that description later. Code-First suggests teams are creating all the code first, then writing API Reference documentation later somehow, either in Google Docs, Spreadsheets, a Content Management System, Postman Collections, or OpenAPI based tooling.
+Stoplight will convert the Postman Collection to an OpenAPI file, and then import all endpoints, models, schemas, and responses into your Stoplight project. This is a great way to get a head start into creating an OpenAPI specification file for existing APIs, improving its documentation, and collaborating with other teams.
```
Comment (heitortsergent) [inline 6.-migrations/postman.md:5]: `import all endpoints,models, schemas, and responses` needs a review, what are the details of the conversion?

**Comment by heitortsergent** [comment]
@pamgoodrich I could use some help in figuring out if/where to move the Postman doc with these updates... I think from looking at the current structure, we can keep it under "Migrations" or move it under "Integrations". What do you think?

**Review by pamgoodrich — APPROVED** [review]
About location.... This doesn't seem like a full migration, but perhaps just an integration. Is that correct? If that's the case, I'd probably put this in integrations.

Also, do we mention anything about data not syncing between the Postman collection and Stoplight after you import to Stoplight? If not, we should. 

Similarly, is there a way to keep the two in sync? Can you reimport and replace? I don't think that works with other imports, but it might be worth asking @falsaffa and @lanemc .

**Comment by heitortsergent** [comment]
@pamgoodrich thanks for all the feedback! I updated the doc and moved it under "Integrations", if you wanna take another look.

File: 6.-migrations/postman.md [hunk 2]
```diff
@@ -1,110 +1,38 @@
...
+Stoplight converts the Postman Collection to an OpenAPI 3.1 file, and then imports all endpoints (paths) into your Stoplight project. This is a great way to create an OpenAPI specification file for existing APIs, improve its documentation, and collaborate with other teams on API design.
 
-If your organiation is moving from code-first to design-first, you might be faced with the prospect of manually creating OpenAPI for every single request and response for every single API, and that is not an exciting prospect, so it's common to try and find shortcuts. 
+For each Postman Collection file you import into a project, a new OpenAPI specification file is created for it. There is currently no way to keep a Postman Collection and a Stoplight API specification in sync. If that's something you're interested in seeing in the product, add a note to the Stoplight [roadmap](https://roadmap.stoplight.io/tabs/7-under-consideration).
```
Comment (heitortsergent) [inline 6.-migrations/postman.md:7]: @pamgoodrich I added this blurb to make it clearer one collection = one new API file, mentioned that they're not kept in sync, and added a link to the roadmap. What do you think?

Comment (pamgoodrich) [reply 6.-migrations/postman.md:7]: That works!

**Review by lanemc — APPROVED** [review]
LGTM! Thank you!!

File: 6.-migrations/postman.md (hunk 2 above)
Comment (falsaffa) [reply 6.-migrations/postman.md:7]: I love the plug here!

**Pull request merged**